service/types.go

```diff
@@ -293,3 +303,33 @@ func (m *accountMetadata) UnmarshalJSON(data []byte) error {
 func has0xPrefix(str string) bool {
 	return len(str) >= 2 && str[0] == '0' && (str[1] == 'x' || str[1] == 'X')
 }
+
+type signedTransactionWrapper struct {
+	SignedTransaction string          `json:"signed_tx"`
+	Currency          *types.Currency `json:"currency,omitempty"`
+}
+
+type signedTransactionWrapperWire struct {
```

Comment:
Why do we need this? It's identical to `signedTransactionWrapper`?

Reply:
Following the convention above